src/torrent/peer/peer_list.cc

```diff
@@ -152,25 +152,25 @@ PeerList::insert_available(const void* al) {
   AvailableList::const_iterator availItr  = m_available_list->begin();
   AvailableList::const_iterator availLast = m_available_list->end();
 
-  for (; itr != last; itr++) {
-    if (!socket_address_key::is_comparable_sockaddr(itr->c_sockaddr()) || itr->port() == 0) {
+  for (const auto& addr : *addressList) {
```

Comment:
You did not remove the above 'itr' and 'last'.

Reply:
I had this done locally. Just never pushed.